packages/manager/modules/pci/src/projects/project/storages/object-storage/users/delete/translations/Messages_fr_FR.json

```
@@ -1,8 +1,8 @@
 {
-  "pci_projects_project_storages_containers_users_delete_title": "Supprimer l'utilisateur",
-  "pci_projects_project_storages_containers_users_delete_label": "Supprimer",
-  "pci_projects_project_storages_containers_users_delete_question": "Êtes-vous sûr·e de vouloir supprimer cet utilisateur des utilisateurs S3 Policies ?",
-  "pci_projects_project_storages_containers_users_delete_success": "L’utilisateur {{ user }} a bien été supprimé de la liste des utilisateurs S3.",
-  "pci_projects_project_storages_containers_users_delete_error": "Une erreur s'est produite lors de la suppression de l'utilisateur : {{message}}",
+  "pci_projects_project_storages_containers_users_delete_title": "Supprimer les credentials d'un utilisateur",
+  "pci_projects_project_storages_containers_users_delete_label": "Confirmer",
+  "pci_projects_project_storages_containers_users_delete_question": "Êtes-vous sûr.e de vouloir supprimer cet utilisateur des S3 Policies?",
```

```suggestion
  "pci_projects_project_storages_containers_users_delete_question": "Êtes-vous sûr(e) de vouloir supprimer cet utilisateur des S3 Policies?",
```
I think this would be more correct.

---

I think both can be valid.